packages/now-cli/src/util/projects/link.ts

```diff
@@ -141,13 +141,25 @@ export async function getLinkedProject(
   }
 
   const spinner = output.spinner('Retrieving project…', 1000);
-  let org: Org | null;
-  let project: Project | ProjectNotFound | null;
+  let org: Org | null = null;
+  let project: Project | ProjectNotFound | null = null;
   try {
     [org, project] = await Promise.all([
       getOrgById(client, link.orgId),
       getProjectByIdOrName(client, link.projectId, link.orgId),
     ]);
+  } catch (err) {
+    if (err?.status === 403) {
+      spinner();
+      throw new NowBuildError({
+        message: `Could not retrieve Project Settings. To link your project, remove the .vercel directory and deploy again.`,
+        code: 'PROJECT_UNAUTHORIZED',
+        link: 'https://vercel.link/cannot-load-project-settings',
+      });
```

Comment:
I think the API should be updated instead since we can return these same 3 properties.

Then you can use `output.error(err)` instead.

Reply:
I don't think so. The API is returning the correct response (403), we're simply adding a special handling in this case.

It would be wrong to add this handling in the API instead, where the response is generic.

Reply:
I suppose the message itself is only relevant for the CLI, so it seems good 👍

Reply:
However, the link https://vercel.link/cannot-load-project-settings doesnt seem to have any section in the FAQ https://vercel.com/docs/v2/platform/frequently-asked-questions#cannot-load-project-settings

Reply:
Docs have been merged!